lib/cron/cache-pools.ts

```diff
@@ -15,40 +24,36 @@ const handler: ScheduledHandler = async (event: EventBridgeEvent<string, void>)
 
   const s3 = new S3()
 
-  await Promise.all(
-    _.map(chainProtocols, async ({ protocol, chainId, provider }) => {
-      log.info(`Getting pools for ${protocol} on ${chainId}`)
+  log.info(`Getting pools for ${protocol} on ${chainId}`)
 
-      let pools
-      try {
-        pools = await provider.getPools()
-      } catch (err) {
-        log.error({ err }, `Failed to get pools for ${protocol} on ${chainId}`)
-        throw new Error(`Failed to get pools for ${protocol} on ${chainId}`)
-      }
+  let pools
+  try {
+    pools = await provider.getPools()
+  } catch (err) {
+    log.error({ err }, `Failed to get pools for ${protocol} on ${chainId}`)
+    throw new Error(`Failed to get pools for ${protocol} on ${chainId}`)
```

Comment:
Because we set a lower timeout than 15 mins if this fails should it no longer post in #eng-ops? Should we add a CloudWatch metric here to log?

Reply:
This will post to eng ops since it throws

Reply:
So in the future would we want to just log these error as a cloudwatch metric instead of throwing an error? Then we could have an alarm that says if we get X error metrics over Y amount of time then page?

Reply:
oh jk this is a categorization issue. ignore